### DISCUSSION TITLE
Creating forms (SQL)

### DISCUSSION THREAD

**Discussion by Hercka** [Budibase/budibase#9854] [Help]
Dears,

at first sorry for my english- it's not my native language.
I'm beginner user of SQL and BudiBase and i want to prepare tool for my coworkers for putting data using forms.
I spent many hours to solve it (i tried many configuration) but finally i stuck.

My SQL db is very simple (initial version, will be extended):
![Przechwytywanie](https://user-images.githubusercontent.com/66530684/222277925-59502415-1775-471c-8920-53860227b40c.PNG)

My form should be looks like that:
1. Employee no. (db: Employees)
2. Card type. (db: card_types)
3. income  of selected employee (from field 1)
4. save button 
 

i'm wondering about 2 things:

1. how can i load a Name and surname after chosing employee number by option picker? (text box below option picker)
2.  Can i  add data to 2 tables: 1-  employees by selected income and cards by employee and card type) using one save button? Now i've form inside data picker (cards) and i can't add income option picker.... :(

Thanks in advance!

![2](https://user-images.githubusercontent.com/66530684/222281702-e8681743-8b71-4dc8-8535-e0b0c039c941.PNG)

**Comment by melohagan** [comment]
Hey @Hercka 

Apologies for the late reply! 

You could have a look at this docs page here: https://docs.budibase.com/docs/filter-by-relationships

You can also save data to two tables using one button:
 - https://docs.budibase.com/docs/form-steps
 - https://docs.budibase.com/docs/chaining-actions#example-creating-and-linking-two-table-entries-in-one-form
